mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -301,3 +335,12 @@ def _download_from_cloud(self, remote_file_path, local_path):
             local_path: Local path to download file to.
 
         """
+
+    @abstractmethod
```

Comment:
If there's no-op in presigned_url_artifact_repo.py, can we remove the `@abstractmethod`?